package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "slimerjs-core",
+  "version": "1.0.0",
```

Comment:
no just use the version of SlimerJS 0.10

Firefox is an otherwise unrelated system dependency